src/reynir_correct/errtokenizer.py

```diff
@@ -1138,7 +1144,10 @@ def get() -> CorrectToken:
                 and token.txt == next_token.txt
                 and token.kind == TOK.WORD
             ):
-                if token.txt.lower() in AllowedMultiples.SET:
+                if (
+                    token.txt.lower() in AllowedMultiples.SET
+                    and "C004/w" not in ignore_rules
+                ):
```

Comment:
Since the if-statement leads to two operatons: next_token.set_error() and yield token, it needs to be double-checked that the presence of the ignore_rule does not change the code behaviour in ways additional to the error reporting itself.

Reply:
Fixed according to the isolation policy discussed in other comments